### PR TITLE
#10747: Update Teams AI nodejs sample links

### PR DESCRIPTION
**Pull request by corinagum**
Fix: #10747
Update node js samples to updated directory

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/Teams conversational AI/teams-conversation-ai-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c70eed051a1f69cf255ca21bb245324282519a21/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fteams-conversation-ai-overview.md) | 92 | 33 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6187ae7d-5a52-4b24-8ec1-66c39440a3e0) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by corinagum** [comment]
@michaelmaillot could you review?

**Review by Wajeed-msft — DISMISSED** [review]

**Comment by michaelmaillot** [comment]
Hi @corinagum,

Thanks for mentioning me, but as my PR has been merged (#10367), I don't think I'm supposed to review yours (even if it sounds good to me 😉).

File: msteams-platform/bots/how-to/Teams conversational AI/teams-conversation-ai-overview.md [hunk 1]
```diff
@@ -116,39 +116,38 @@ Vector databases are a new type of database designed to store vectors and enable
...
+| Food ordering assistant                | This example shows how to create a conversational assistant that uses tools to call actions in your bots code. It's a food ordering assistant for a fictional restaurant called The Pub and is capable of complex interactions with the user as it takes their order. | [View](https://github.com/microsoft/teams-ai/tree/main/dotnet/samples/06.assistants.b.orderBot)             | [View](https://github.com/microsoft/teams-ai/tree/main/js/samples/04.ai-apps/e.assistants-orderBot)                    |
 
 ## Next step
 
-> [!div class="nextstepaction"]
-> [Teams AI library capabilities](how-conversation-ai-core-capabilities.md)
+> [!div class="nextstepaction"] > [Teams AI library capabilities](how-conversation-ai-core-capabilities.md)
```
[the remark this answers is not in the document]
Comment (v-sdhakshina) [reply msteams-platform/bots/how-to/Teams conversational AI/teams-conversation-ai-overview.md:153]: @corinagum request you to check and update this suggestion.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/Teams conversational AI/teams-conversation-ai-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/05f75e2e051b9b7998944339f18de77bcf203627/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fteams-conversation-ai-overview.md) | 91 | 34 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9c451b98-e7b4-4848-842e-8555b8f4d72c) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-sdhakshina — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/Teams conversational AI/teams-conversation-ai-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/076c85f61731b8f80a24f450786652038d350c57/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fteams-conversation-ai-overview.md) | 91 | 34 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/660074be-a562-4dca-837d-573ba82a6230) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-deabira — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/Teams conversational AI/teams-conversation-ai-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/eb057f2e63a2ca71a52cf86dd1d602f83882b3ef/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fteams-conversation-ai-overview.md) | 91 | 34 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ed51e738-238f-406d-befd-67448a8fefcf) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]

**Pull request merged**